logging every time an event is added/updated

sarojkandel12 opened the pull request.

sarojk-zg committed:
logging every time an event is added/updated

sarojkandel12 commented:
@toumorokoshi This commit adds info logs to event updates. Could you review and publish a new version if it looks good?

toumorokoshi commented on tycho/routes/event.py:
```diff
@@ -137,6 +140,7 @@
     :return: the stored event
     """
     await request.app["db"].event.save(event)
+    LOG.info(event.to_primitive())
```
could this be a configuration parameter? just thinking people may not want to log every event normally.

Then again the volume is pretty low.

sarojk-zg committed:
adding a configuration parameter for logging events

There are cases where we might want to log events whenever they are added/updated to Tycho.
This commit adds a configuration option that logs events by default, but can be overriden to not log updates
